clients/instance/ibm-pi-helper.go

```diff
@@ -21,5 +30,10 @@ func NewIBMPIClient(ctx context.Context, sess *ibmpisession.IBMPISession, cloudI
 		session:         sess,
 		cloudInstanceID: cloudInstanceID,
 		ctx:             ctx,
+
+		dhcpRequest:           sess.Power.PCloudServicedhcp,
```

Comment:
Yussuf also brought up a good point on duplicate information in this object. Is this part of the suggested style?